#!/usr/bin/env python
PACKAGE = "obstacle_detection"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("voxel_grid_32_high",		double_t, 0, "voxel_grid_32_high",    	  0.15, 0.05, 0.4);
gen.add("voxel_grid_32_low",		double_t, 0, "voxel_grid_32_low",    	  0.4, 0.3, 0.6);
gen.add("distance_threshold",		double_t, 0, "distance_threshold",    	  0.1, 0.05, 0.5);
gen.add("filter_ground_range",		double_t, 0, "filter_ground_range",    	  0.2, 0.05, 0.5);
gen.add("MinClusterSize",		int_t, 0, "MinClusterSize",    	  10, 5, 40);
gen.add("MaxClusterSize",		int_t, 0, "MaxClusterSize",    	  10000, 1000, 100000);
gen.add("ClusterTolerance",		double_t, 0, "ClusterTolerance",    	  0.2, 0.05, 0.5);
gen.add("theta",                        double_t, 0, "theta",    	  3, 1, 15);



exit(gen.generate(PACKAGE, "obstacle_detection", "Cloud_transform"))
